micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/common/response/receive/HeadersSetting.groovy

```diff
@@ -125,4 +125,23 @@ interface HeadersSetting<T> extends Executable<T> {
      */
     HeadersSetting<T> headers(HttpHeaders httpHeaders)
 
+    /**
+     * Sets value for the {@link HttpHeaders#AUTHORIZATION} header
+     *
+     * @param authorization
+     *
+     * @return itself
+     */
+    HeadersSetting<T> authorization(String authorization)
+
+    /**
+     * Sets basic authorization value for the {@link HttpHeaders#AUTHORIZATION} header
+     *
+     * @param username
+     * @param password
+     *
+     * @return itself
+     */
+    HeadersSetting<T> basicAuthorization(String username, String password)
```

Comment:
The correct name is BASIC *authentication*, despite HTTP header being named `Authorization` (sic!), see: http://en.wikipedia.org/wiki/Basic_access_authentication